src/NumericalAlgorithms/Interpolation/PredictedZeroCrossing.cpp

```diff
@@ -16,4 +16,15 @@ double predicted_zero_crossing_value(const std::vector<double>& x_values,
   return -coefficients[0]/coefficients[1];
 }
 
+std::vector<double> predicted_zero_crossing_value(
```

Comment:
Not a change request necessarily, but I'm wondering why you return `std::vector<double>` instead of `DataVector`?